spec/javascripts/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -125,7 +125,6 @@ describe('document-capture/components/acuant-capture', () => {
 
       const button = await findByText('doc_auth.buttons.upload_picture');
       expect(button.classList.contains('usa-button--outline')).to.be.true();
-      expect(console).to.have.loggedError(/^Error: Could not load script:/);
```

Comment:
does this not work because `console` is the real value and not the spy anymore?

Reply:
> does this not work because `console` is the real value and not the spy anymore?

Previously, we were letting JSDOM try to actually (virtually) load the script, and in this test scenario we were expecting that to fail, at which point JSDOM would [log to the console](https://github.com/jsdom/jsdom/blob/04f6c13f4a4d387c7fc979b8f62c6f68d8a0c639/lib/jsdom/browser/resources/per-document-resource-loader.js#L38-L42), and we needed to allow for that to happen with `loggedError`.

Since we're no longer letting JSDOM load the scripts, the log no longer happens.